mvc_controller.hpp

```diff
@@ -463,6 +466,8 @@ class MvcController final
     MvcModel& model_;
     MvcView const& view_;
 
+    std::map<std::string, std::vector<wxString>> itemboxes_cache_;
```

Comment:
```suggestion
    std::unordered_map<std::string, std::vector<wxString>> itemboxes_cache_;
```
(we also would need to include the appropriate header, of course).

AFAICS we don't need the keys to be sorted here, so using `unordered_map` might result in a slightly better performance.